native_client/ctcdecode/build_archive.py

```diff
@@ -82,6 +102,15 @@ def build_one(file):
         )
         print(cmd)
         subprocess.check_call(shlex.split(cmd))
+    elif sys.platform.startswith('win'):
+        obj_files = [s for s in obj_files if s != None]
+        cmd = '"lib.exe" /OUT:"{outfile}" {infiles} /MACHINE:X64 /NOLOGO'.format(
```

Comment:
This is not working for me, system complains it cannot find `lib.exe`

Reply:
```
"lib.exe" /OUT:"third_party.lib" temp_build/temp_build/../kenlm/util/float_to_string.o temp_build/temp_build/../kenlm/util/integer_to_string.o temp_build/temp_build/../kenlm/util/bit_packing.o temp_build/temp_build/../kenlm/util/murmur_hash.o temp_build/temp_build/../kenlm/util/ersatz_progress.o temp_build/temp_build/../kenlm/util/file_piece.o temp_build/temp_build/../kenlm/util/spaces.o temp_build/temp_build/../kenlm/util/pool.o temp_build/temp_build/../kenlm/util/parallel_read.o temp_build/temp_build/../kenlm/util/scoped.o temp_build/temp_build/../kenlm/util/string_piece.o temp_build/temp_build/../kenlm/util/read_compressed.o temp_build/temp_build/../kenlm/lm/bhiksha.o temp_build/temp_build/../kenlm/lm/lm_exception.o temp_build/temp_build/../kenlm/lm/config.o temp_build/temp_build/../kenlm/lm/binary_format.o temp_build/temp_build/../kenlm/lm/quantize.o temp_build/temp_build/../kenlm/lm/read_arpa.o temp_build/temp_build/../kenlm/lm/model.o temp_build/temp_build/../kenlm/lm/search_hashed.o temp_build/temp_build/../kenlm/lm/trie.o temp_build/temp_build/../kenlm/lm/sizes.o temp_build/temp_build/../kenlm/lm/value_build.o temp_build/temp_build/../kenlm/lm/virtual_interface.o temp_build/temp_build/../kenlm/util/double-conversion/bignum-dtoa.o temp_build/temp_build/../kenlm/lm/trie_sort.o temp_build/temp_build/../kenlm/util/double-conversion/cached-powers.o temp_build/temp_build/../kenlm/util/double-conversion/bignum.o temp_build/temp_build/../kenlm/util/double-conversion/diy-fp.o temp_build/temp_build/../kenlm/util/double-conversion/fast-dtoa.o temp_build/temp_build/../kenlm/util/double-conversion/double-conversion.o temp_build/temp_build/../kenlm/util/double-conversion/fixed-dtoa.o temp_build/temp_build/../kenlm/util/double-conversion/strtod.o temp_build/temp_build/../kenlm/lm/vocab.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/compat.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/flags.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/mapped-file.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/fst.o temp_build/temp_build/../kenlm/util/usage.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/properties.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/symbol-table-ops.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/util.o temp_build/temp_build/../kenlm/util/mmap.o temp_build/temp_build/../kenlm/util/file.o temp_build/temp_build/../kenlm/util/exception.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/symbol-table.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/weight.o temp_build/temp_build/../kenlm/lm/search_trie.o temp_build/temp_build/third_party/openfst-1.6.9-win/src/lib/fst-types.o /MACHINE:X64 /NOLOGO
Traceback (most recent call last):
  File "./setup.py", line 64, in <module>
    maybe_rebuild(KENLM_FILES, third_party_build, build_dir)
  File "./setup.py", line 52, in maybe_rebuild
    debug=debug)
  File "C:\builds\tc-workdir\DeepSpeech\ds\native_client\ctcdecode\build_archive.py", line 113, in build_archive
    subprocess.check_call(shlex.split(cmd))
  File "C:\builds\tc-workdir\pyenv-root\versions\3.5.4_ucs2\tools\lib\subprocess.py", line 266, in check_call
    retcode = call(*popenargs, **kwargs)
  File "C:\builds\tc-workdir\pyenv-root\versions\3.5.4_ucs2\tools\lib\subprocess.py", line 247, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\builds\tc-workdir\pyenv-root\versions\3.5.4_ucs2\tools\lib\subprocess.py", line 676, in __init__
    restore_signals, start_new_session)
  File "C:\builds\tc-workdir\pyenv-root\versions\3.5.4_ucs2\tools\lib\subprocess.py", line 957, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
make: *** [Makefile:44: bindings] Error 1
make: Leaving directory '/c/builds/tc-workdir/DeepSpeech/ds/native_client/ctcdecode'
```
https://community-tc.services.mozilla.com/tasks/OPvykPaMR_Osfp02xibGRA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FOPvykPaMR_Osfp02xibGRA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L2492-2510

Reply:
Hm maybe I have an idea ...

Reply:
Ok, that's fixed.